netket/nn/symmetric_linear.py

```diff
@@ -126,7 +126,7 @@ def __call__(self, x: Array) -> Array:
             "kernel", self.kernel_init, (self.features, self.n_sites), self.dtype
         )
 
-        if self.mask:
+        if not self.mask is None:
```

Comment:
python would like you to use
```suggestion
        if self.mask is not None:
```

Reply:
Okay, that's in four places so I'll fix it on my end